### PR TITLE
test_all: Cleanup loop device

### PR DESCRIPTION
**Pull request by lsgunth**
Allow choosing the disk type for the entire test (loop, ram or dev).

Also allow running both ram and loop sequentially.

### CODE REVIEW

File: test_all [hunk 1]
```diff
@@ -54,29 +52,56 @@ TEST_ENVS = {
...
     os.environ["QUIET"] = "y"
 
+    if args.disk_type is None:
+        if "DEVS" in os.enivron:
+            args.disk_type = "devs"
+        else:
+            args.disk_type = "ram"
+
+    if "DEVS" in os.environ and args.disk_type != "devs":
+        sys.exit("If DEVS is set in the environment, --disk-type must be 'devs'")
```
Comment (dlsloan) [inline test_all]: Why? it's convenient to set DEVS once, and then just select the DISK_TYPE as needed when running things manually

Comment (lsgunth) [reply test_all]: Yes, good point. I think I was worried that DEVS in the enviroment might change the test, but it should be fine....

Though I should check DISK_TYPE to ensure it matches....

**Review by dlsloan — APPROVED** [review]
Looks good

**Comment by lsgunth** [comment]
Oops, you were too quick. I made some changes please look again.

**Comment by dlsloan** [comment]
Gave it another look, still looks good to me

**Pull request merged**